core/services/relay/evm/chain_reader.go

```diff
@@ -178,7 +179,12 @@ func (cr *chainReader) Close() error {
 func (cr *chainReader) Ready() error { return nil }
 
 func (cr *chainReader) HealthReport() map[string]error {
-	return map[string]error{cr.Name(): nil}
+	report := map[string]error{
+		cr.Name(): cr.Healthy(),
```

Comment:
Ideally the CR Healthy() implementation should append all reports from underlying components like ht, lp, client, etc.
But I assume for now you are only worried about making FV work.

Reply:
Good point. Initially, I thought HT and the Client did not support reports, but it turns out that I've checked HT's interface instead of implementation. 
Added propagation of HT reports. Created a[ticket](https://smartcontract-it.atlassian.net/browse/BCFR-1092) to also implement it for client.